runtime/domain.c

```diff
@@ -1125,8 +1125,8 @@ static void handover_ephemerons(caml_domain_state* domain_state)
       domain_state->ephe_info->live == 0)
     return;
 
-  todo_tail = caml_bias_ephe_list(domain_state->ephe_info->todo, (struct domain*)NULL);
-  live_tail = caml_bias_ephe_list(domain_state->ephe_info->live, (struct domain*)NULL);
+  todo_tail = caml_bias_ephe_list(domain_state->ephe_info->todo);
+  live_tail = caml_bias_ephe_list(domain_state->ephe_info->live);
   caml_add_orphaned_ephe(domain_state->ephe_info->todo, todo_tail,
```

Comment:
It would be better to change the interface to take just `domain_state->ephe_info->todo` and `domain_state->ephe_info->live` and `caml_add_orphaned_ephe` appends the lists to the orphaned lists. It would also be useful to rename this function as `caml_add_to_orphaned_ephe_list`.